pandas/core/groupby/generic.py

```diff
@@ -973,6 +973,14 @@ def fillna(
         mouse  0.0
         dtype: float64
         """
+        warnings.warn(
```

Comment:
Could you add a `.. deprecated:: 2.2.0` directive near the top of the docstring?

Reply:
Thanks - I also removed the other deprecations from the docstring. Seemed like if the entire method was deprecated we didn't need notes on individual parameters. Can restore if desired.

Reply:
I think that cleanup makes sense. Thanks!